src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_FIGURE_PROPERTY_VALUE = 30;
+    private static final int MIN_FIG_VALUE = 1;
+    private static final double DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.getValue();
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Circle defaultFigure = new Circle();
```

Comment:
You have to create a new object in the `getDefaultFigure` method. It may cause an error